frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -21,7 +22,14 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
   <FieldArray
     name={name}
     render={({ push, remove, form }) => {
-      const fieldValue = _.get(form.values, name, []);
+      const fieldValues = _.get(form.values, name, []);
+      // Add a unique key to the object for keyed react list rendering
+      const keyedFieldValues = fieldValues.map((value) => {
+        if (!('uniqueKeyId' in value)) {
+          value.uniqueKeyId = uuid();
```

Comment:
For the public record - I still feel this is not the best way to do this - I just don't have a better idea off the top of my head. I feel there needs to be some external creation of an id - but @rohitkrai03 and I had a conversation about it and feel there is a lot of factors that go into this - and the internal functionality of this component being known externally is subject to coupling. 

Since the external API of this component is to allow for multiple "on the fly" rows of the simple elements provided by the wrapping component, the ownership of "handling multiple rows" falls on this component and not the calling code.